rstan/rstan/R/zzz.R

```diff
@@ -18,10 +18,19 @@
 rstan_load_time <- as.POSIXct("1970-01-01 00:00.00 UTC")
 RNG <- 0
 OUT <- 0
+stanc_ctx <- V8::v8()
 
 tbbmalloc_proxyDllInfo <- NULL
 
 .onLoad <- function(libname, pkgname) {
+  assignInMyNamespace("stanc_ctx",  value= V8::v8())
+  stanc_js <- system.file("stanc.js", package = "rstan")
+  if (!file.exists(stanc_js)) {
+    warning(paste0("Default stancjs compiler not found, ",
+                   "downloading the current version from github."))
+    stanc_js <- "https://github.com/stan-dev/stanc3/releases/download/v2.27.0/stanc.js"
+  }
+  rstan:::stanc_ctx$source(stanc_js)
```

Comment:
There's no need for the internal call.
```suggestion
  stanc_ctx$source(stanc_js)
```

Reply:
During debugging or something like that could a global `stanc_ctx` be used instead of the `stanc_ctx` in the namespace? If not then I'm fine with removing the internal call

Reply:
I'm not sure if CRAN allows calling internal functions (i.e., using `:::`), and it's ok if we can overwrite the `v8` call since `stanc_ctx` name can't be confusing.